generator/src/lib.rs

```diff
@@ -919,11 +919,14 @@ fn generate_function_pointers<'a>(
             let name = &cmd.proto.name;
             let type_name = format_ident!("PFN_{}", name);
 
-            let function_name_c = if let Some(alias_name) = aliases.get(name) {
-                alias_name.to_string()
+            // We might need to generate a function pointer for an extension, where we are given the original
+            // `cmd` and a rename back to the extension alias (typically with vendor suffix) in `rename_commands`:
```

Comment:
is the trailing colon meant ?

Reply:
I typically use trailing colons in comments to indicate that it relates to the next line, specifically. See other instances of `//.*:$` within the codebase.